lib/build.js

```diff
@@ -21,26 +21,34 @@ module.exports = {
       enum: [ 'Toggle', 'Keep Visible', 'Show on Error' ],
       order: 1
     },
+    panelOrientation: {
+      title: 'Panel Orientation',
+      description: 'Set where the build panel should be.',
+      type: 'string',
+      default: 'Bottom',
+      enum: [ 'Top', 'Bottom', 'Left', 'Right' ],
```

Comment:
I'd love to see an updated PR!

I also want to see a simple test which tests that setting the orientation actually attaches the panel at different locations. Take other tests as inspiration. It is probably easiest to check the classes on the build panel (I'm guessing Atom adds different classes depending if it is a `TopPanel` or a `BottomPanel`).